src/client/android.ts

```diff
@@ -129,6 +177,7 @@ export class AndroidDevice extends ChannelOwner<channels.AndroidDeviceChannel, c
   async close() {
     return this._wrapApiCall('androidDevice.close', async () => {
       await this._channel.close();
+      this.emit(Events.AndroidDevice.Close);
```

Comment:
I wonder how this works if I disconnect the cable?

Reply:
we'll figure it with time...